rasa_nlu/utils/md_to_json.py

```diff
@@ -15,10 +15,16 @@
 intent_regex = re.compile('##\s*intent:(.+)')
 synonym_regex = re.compile('##\s*synonym:(.+)')
 example_regex = re.compile('\s*-\s*(.+)')
+comment_regex = re.compile('\s*<!--.*-->\s*')
 
 INTENT_PARSING_STATE = "intent"
 SYNONYM_PARSING_STATE = "synonym"
 
+def strip_comments(comment_regex,text): 
+    """ Removes comments defined by the argument 'comment_regex' from the 
```

Comment:
no space at the beginning. Function comments should either be single line or multi line with a whitespace, e.g.
lets do ```"""Remove comments defined by `comment_regex` from `text`. """```

Reply:
Okay, so I should  1. fix it,  2. commit and then 3. push again, right ?
or is there a quicker way to do this?